raiden_contracts/contract_manager.py

```diff
@@ -146,13 +147,25 @@ def gas_measurements(version: Optional[str] = None) -> Dict[str, int]:
         return json.load(gas_file)
 
 
+class ContractDevEnvironment(enum.Enum):
+    DEMO = "demo"
+    UNSTABLE = "unstable"
+
+
 def contracts_deployed_path(
-    chain_id: ChainID, version: Optional[str] = None, services: bool = False
+    chain_id: ChainID,
+    version: Optional[str] = None,
+    services: bool = False,
+    development_environment: ContractDevEnvironment = ContractDevEnvironment.DEMO,
 ) -> Path:
     """Returns the path of the deplolyment data JSON file."""
     data_path = contracts_data_path(version)
     chain_name = ID_TO_CHAINNAME[chain_id] if chain_id in ID_TO_CHAINNAME else "private_net"
 
+    if development_environment == ContractDevEnvironment.UNSTABLE:
+        assert chain_name == "goerli", "Unstable contracts only available on goerli"
+        chain_name += "_unstable"
```

Comment:
with the `assert` above, why not be upfront?
```suggestion
        chain_name = "goerli_unstable"
```